tests/test_validators_jsonschema.py

```diff
@@ -43,13 +43,21 @@ class AdditionalItems(SchemaTest):
     data_tests = [
         DataTest(
             name="additionalItems as schema, additional items match schema",
-            schema={"items": [{}], "additionalItems": {"type": "integer"}},
+            schema={
+                "$schema": "http://json-schema.org/draft-07/schema#",
+                "items": [{}],
+                "additionalItems": {"type": "integer"},
+            },
```

Comment:
`jsonschema` 4 now [supports newer drafts of JSON Schema](https://github.com/Julian/jsonschema/blob/main/CHANGELOG.rst#v400), so as I am **not** an expert in JSON Schemas, I opted to specify the previous draft (the one taken as default in `jsonschema` 3) instead of testing the new stuff of the newer drafts.

Many tests were failing due to the [situation described in this `jsonschema` issue](https://github.com/Julian/jsonschema/issues/852#issuecomment-933376870).

I think (a) testing new JSON Schema drafts is the role of `jsonschema`, not Spidermon's;
and (b) if we want to add tests specific to a given JSON Schema draft, this should be the focus of a new PR.